src/util/notify.cpp

```diff
@@ -29,7 +29,7 @@ namespace big::notify
 				return;
 
 			if (g.reactions.crash.notify)
-				g_notification_service.push_error("Protections", std::format("Blocked {} crash from {}", crash, player->get_name()));
+				g_notification_service.push_error("Protections", std::vformat("BLOCKED_CRASH"_T.data(), std::make_format_args(crash,player->get_name())));
```

Comment:
`Protections` should already be in the translation repo called `PROTECTIONS`.

Should be an easy fix as the other guy said.